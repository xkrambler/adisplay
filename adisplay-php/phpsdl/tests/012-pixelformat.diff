004+ /usr/bin/php7.0: symbol lookup error: /data/dev/accord_display8th/adisplay/phpsdl/modules/sdl.so: undefined symbol: Z_TYPE_PP
004- int(0)
005- = PixelFormat
006- = PixelFormatPalette
007- int(0)
008- = Result
009- object(SDL_PixelFormat)#%d (16) {
010-   ["format"]=>
011-   int(303039488)
012-   ["BitsPerPixel"]=>
013-   int(4)
014-   ["BytesPerPixel"]=>
015-   int(1)
016-   ["Rmask"]=>
017-   int(0)
018-   ["Gmask"]=>
019-   int(0)
020-   ["Bmask"]=>
021-   int(0)
022-   ["Amask"]=>
023-   int(0)
024-   ["Rloss"]=>
025-   int(8)
026-   ["Gloss"]=>
027-   int(8)
028-   ["Bloss"]=>
029-   int(8)
030-   ["Aloss"]=>
031-   int(8)
032-   ["Rshift"]=>
033-   int(0)
034-   ["Gshift"]=>
035-   int(0)
036-   ["Bshift"]=>
037-   int(0)
038-   ["Ashift"]=>
039-   int(0)
040-   ["palette"]=>
041-   object(SDL_Palette)#%d (4) {
042-     ["ncolors"]=>
043-     int(16)
044-     ["version"]=>
045-     int(2)
046-     ["refcount"]=>
047-     int(2)
048-     ["colors"]=>
049-     array(16) {
050-       [0]=>
051-       object(SDL_Color)#%d (4) {
052-         ["r"]=>
053-         int(0)
054-         ["g"]=>
055-         int(0)
056-         ["b"]=>
057-         int(0)
058-         ["a"]=>
059-         int(0)
060-       }
061-       [1]=>
062-       object(SDL_Color)#%d (4) {
063-         ["r"]=>
064-         int(17)
065-         ["g"]=>
066-         int(17)
067-         ["b"]=>
068-         int(17)
069-         ["a"]=>
070-         int(17)
071-       }
072-       [2]=>
073-       object(SDL_Color)#%d (4) {
074-         ["r"]=>
075-         int(34)
076-         ["g"]=>
077-         int(34)
078-         ["b"]=>
079-         int(34)
080-         ["a"]=>
081-         int(34)
082-       }
083-       [3]=>
084-       object(SDL_Color)#%d (4) {
085-         ["r"]=>
086-         int(51)
087-         ["g"]=>
088-         int(51)
089-         ["b"]=>
090-         int(51)
091-         ["a"]=>
092-         int(51)
093-       }
094-       [4]=>
095-       object(SDL_Color)#%d (4) {
096-         ["r"]=>
097-         int(68)
098-         ["g"]=>
099-         int(68)
100-         ["b"]=>
101-         int(68)
102-         ["a"]=>
103-         int(68)
104-       }
105-       [5]=>
106-       object(SDL_Color)#%d (4) {
107-         ["r"]=>
108-         int(85)
109-         ["g"]=>
110-         int(85)
111-         ["b"]=>
112-         int(85)
113-         ["a"]=>
114-         int(85)
115-       }
116-       [6]=>
117-       object(SDL_Color)#%d (4) {
118-         ["r"]=>
119-         int(102)
120-         ["g"]=>
121-         int(102)
122-         ["b"]=>
123-         int(102)
124-         ["a"]=>
125-         int(102)
126-       }
127-       [7]=>
128-       object(SDL_Color)#%d (4) {
129-         ["r"]=>
130-         int(119)
131-         ["g"]=>
132-         int(119)
133-         ["b"]=>
134-         int(119)
135-         ["a"]=>
136-         int(119)
137-       }
138-       [8]=>
139-       object(SDL_Color)#%d (4) {
140-         ["r"]=>
141-         int(136)
142-         ["g"]=>
143-         int(136)
144-         ["b"]=>
145-         int(136)
146-         ["a"]=>
147-         int(136)
148-       }
149-       [9]=>
150-       object(SDL_Color)#%d (4) {
151-         ["r"]=>
152-         int(153)
153-         ["g"]=>
154-         int(153)
155-         ["b"]=>
156-         int(153)
157-         ["a"]=>
158-         int(153)
159-       }
160-       [10]=>
161-       object(SDL_Color)#%d (4) {
162-         ["r"]=>
163-         int(170)
164-         ["g"]=>
165-         int(170)
166-         ["b"]=>
167-         int(170)
168-         ["a"]=>
169-         int(170)
170-       }
171-       [11]=>
172-       object(SDL_Color)#%d (4) {
173-         ["r"]=>
174-         int(187)
175-         ["g"]=>
176-         int(187)
177-         ["b"]=>
178-         int(187)
179-         ["a"]=>
180-         int(187)
181-       }
182-       [12]=>
183-       object(SDL_Color)#%d (4) {
184-         ["r"]=>
185-         int(204)
186-         ["g"]=>
187-         int(204)
188-         ["b"]=>
189-         int(204)
190-         ["a"]=>
191-         int(204)
192-       }
193-       [13]=>
194-       object(SDL_Color)#%d (4) {
195-         ["r"]=>
196-         int(221)
197-         ["g"]=>
198-         int(221)
199-         ["b"]=>
200-         int(221)
201-         ["a"]=>
202-         int(221)
203-       }
204-       [14]=>
205-       object(SDL_Color)#%d (4) {
206-         ["r"]=>
207-         int(238)
208-         ["g"]=>
209-         int(238)
210-         ["b"]=>
211-         int(238)
212-         ["a"]=>
213-         int(238)
214-       }
215-       [15]=>
216-       object(SDL_Color)#%d (4) {
217-         ["r"]=>
218-         int(255)
219-         ["g"]=>
220-         int(255)
221-         ["b"]=>
222-         int(255)
223-         ["a"]=>
224-         int(255)
225-       }
226-     }
227-   }
228- }
229- = RGB
230- int(8)
231- int(12)
232- NULL
233- int(85)
234- int(85)
235- int(85)
236- NULL
237- int(170)
238- int(170)
239- int(170)
240- int(170)
241- = Free
242- = Done