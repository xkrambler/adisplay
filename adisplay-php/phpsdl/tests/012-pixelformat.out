= Palette
= Color
= PaletteColors
/usr/bin/php7.0: symbol lookup error: /data/dev/accord_display8th/adisplay/phpsdl/modules/sdl.so: undefined symbol: Z_TYPE_PP